### PR TITLE
Feature/gbftriggerbox2

### PR DESCRIPTION
**Pull request by Frickerson**

### CODE REVIEW

File: Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h [hunk 1]
```diff
@@ -1,21 +1,22 @@
-#pragma once
+// Fill out your copyright notice in the Description page of Project Settings.
```
Comment (TheEmidee) [inline Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h:1]: remove

File: Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h [hunk 2]
```diff
@@ -1,21 +1,22 @@
-#pragma once
+// Fill out your copyright notice in the Description page of Project Settings.
 
-#include <CoreMinimal.h>
-#include <Engine/TriggerBox.h>
+#pragma once
 
+#include "CoreMinimal.h"
```
Comment (TheEmidee) [inline Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h]: <>

File: Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h (hunk 1 above)
Comment (TheEmidee) [inline Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h]: Remove

File: Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h [hunk 3]
```diff
@@ -1,21 +1,22 @@
-#pragma once
+// Fill out your copyright notice in the Description page of Project Settings.
 
-#include <CoreMinimal.h>
-#include <Engine/TriggerBox.h>
+#pragma once
 
+#include "CoreMinimal.h"
+#include "GameFramework/Actor.h"
```
Comment (TheEmidee) [inline Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h]: <>

**Review by TheEmidee — CHANGES_REQUESTED** [review]

File: Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h (hunk 3 above)
Comment (Frickerson) [reply Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h]: Done.

File: Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h (hunk 2 above)
Comment (Frickerson) [reply Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h]: Done.

File: Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h (hunk 1 above)
Comment (Frickerson) [reply Source/GameBaseFramework/Public/Gameplay/GBFTriggerBox.h]: Done.

**Review by TheEmidee — APPROVED** [review]

**Pull request merged**